infinitest-eclipse/src/main/java/org/infinitest/eclipse/prefs/SwtColorFieldEditor.java

```diff
@@ -0,0 +1,33 @@
+package org.infinitest.eclipse.prefs;
+
+import org.eclipse.jface.preference.*;
+import org.eclipse.swt.*;
+import org.eclipse.swt.widgets.*;
+
+public class SwtColorFieldEditor extends ComboFieldEditor {
```

Comment:
The problem is that ColorFaceEditor returns an RGB and creating a color thats usable for swt gui elements from RGB is complicated and weird and involves colors being a system resource that have to be freed after use.
I wanted to save myself that hassle and therefore created an editor where you can choose from SWT's pre-defined colors (constants on the SWT class) which are managed by SWT itself.